apps/know-me/src/styles/_utilities.scss

```diff
@@ -0,0 +1,89 @@
+.cursor--pointer {
+  cursor: pointer;
+}
+
+.display--block {
+  display: block;
+}
+
+.object-fit {
+  &--cover {
+    object-fit: cover;
+  }
+
+  &--contain {
+    object-fit: contain;
+  }
+}
+
+.position--relative {
+  position: relative;
+}
+
+.text {
+  &--center {
+    text-align: center;
+  }
+
+  &--right {
+    text-align: right;
+  }
+
+  &--decoration {
+    text-decoration: line-through;
+  }
+
+  &--uppercase {
+    text-transform: uppercase;
+  }
+
+  &--capitalize {
+    text-transform: capitalize;
+  }
+
+  &--decoration-none {
+    text-decoration: none;
+  }
+}
+
+.width {
+  &--full {
+    width: 100%;
+  }
+
+  &--inherit {
+    width: inherit;
+  }
+
+  &--fit-content {
+    width: fit-content;
+  }
+
+  &--fill-available {
+    width: -webkit-fill-available;
+  }
+}
+
+.height {
+  &--full {
+    height: 100%;
+  }
+
+  &--fit-content {
+    height: fit-content;
+  }
+}
+
+.border{
+  &--input fieldset {
+    border-radius: 8px;
+  }
+}
+
+.custom-search-input {
```

Comment:
please try to have _overwrite.scss under styles which you can overwrite styling for all the component

Reply:
oh make different style sheet mo?

Reply:
yup like _overwrite.scss but this overwrite styling have to be affected to whole particular components and incase you want to have those affect in one UI you can do it in that UI component stylesheet